docs/resources/contributing.md

```diff
@@ -74,7 +74,10 @@ The author reserves the right to reject any PR that's outside the scope of the p
 
 ## Developing
 
-To set up a local dev environment, [fork the repo](https://github.com/shoelace-style/shoelace/fork) on GitHub, clone it locally, and install its dependencies.
```

Comment:
I'd like to leave local as the first and "default" development strategy. We can add Gitpod below it, but I don't want to come across as recommending a particular service over local dev.

I'm happy to tweak this content myself if you'd rather me do it.